tag_to_header.py

```diff
@@ -241,10 +241,10 @@ def main():
 
 				if tag[o.taglen:] + tag[:o.taglen] in barcode_dict and barcode_dict[tag[o.taglen:] + tag[:o.taglen]] >= 3:
 					dcs_count += 1
-
+ 
 					if o.reduce and tag not in dcs_tags_list:
```

Comment:
even if I'm using a set I left `tag not in` because I don't know if `tag[o.taglen:] + tag[:o.taglen]` could differ w/ the same `tag` ...